### DISCUSSION TITLE
Modal that views and edits the fields of a selected row in a table

### DISCUSSION THREAD

**Discussion by egattulli** [Budibase/budibase#10169] [Help] [answered]
Hi everyone,

I need to create a table that when I click on a row it opens a modal. The modal should display a form with all the fields of the related row selected from the table. The fields should be editable.

How can I achieve this? Thanks in advance!

**Comment by melohagan** [comment, marked as answer]
Hey @egattulli 

Here's a couple of docs pages that should help get you started!

 - https://docs.budibase.com/docs/screens#modal-screen
 - https://docs.budibase.com/docs/url-parameters
 - https://docs.budibase.com/docs/forms
